### ISSUE TITLE
Unhandled errors on MonoBehavior not being registered if Debug is disabled

### ISSUE THREAD

**Issue by lucas-zimerman** [getsentry/sentry-unity#483]
If you run the following script with Debug disabled, Sentry will not log any erros. But once enabled, the error will be registered.
```csharp
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using Sentry;
using System;
using System.Threading.Tasks;

public class NewBehaviourScript : MonoBehaviour
{
    void Start()
    {
        Start2();
    }

    void Start2()    
    {
        var transaction = SentrySdk.StartTransaction(
        "test-transaction-name",
        "test-transaction-operation"
        );

        SentrySdk.ConfigureScope(scope =>scope.Transaction = transaction);
        //user code...
        throw new Exception(" A bug");
        //transaction.Finish(OK);
    }

    // Update is called once per frame
    void Update()
    {
    }
}
```
EDIT: Events were sent but only after restarting the app.

**Comment by bruno-garcia** [comment]
> EDIT: Events were sent but only after restarting the app.

So there's no issue?

**Comment by bruno-garcia** [comment]
Resolved by #484 

Working on getting a release build ready to get published (version 0.9.0)
